apps/condo/domains/marketplace/schema/MarketPriceScope.test.js

```diff
@@ -0,0 +1,302 @@
+            let client, property
+            beforeAll(async () => {
+                [property] = await createTestProperty(admin, organization)
+                client = await makeClient()
+            })
+            test('can\'t create', async () => {
+                await expectToThrowAuthenticationErrorToObj(async () => {
+                    await createTestMarketPriceScope(client, price, property)
+                })
+            })
+
+            test('can\'t update', async () => {
+                await expectToThrowAuthenticationErrorToObj(async () => {
+                    await updateTestMarketPriceScope(client, 'id')
+                })
+            })
+
+            test('can\'t delete', async () => {
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await MarketPriceScope.delete(client, 'id')
+                })
+            })
+
+            test('can\'t read', async () => {
+                await expectToThrowAuthenticationErrorToObjects(async () => {
+                    await MarketPriceScope.getAll(client, {}, { sortBy: ['updatedAt_DESC'] })
+                })
+            })
+        })
+
+        describe('resident', () => {
+            let organization, client, property
+            beforeAll(async () => {
+                [organization] = await createTestOrganization(admin)
+                client = await makeClientWithResidentUser()
+                const unitName = faker.random.alphaNumeric(8)
+                const accountNumber = faker.random.alphaNumeric(8);
+
+                [property] = await createTestProperty(admin, organization)
+                const [resident] = await createTestResident(admin, client.user, property, {
+                    unitName,
+                })
+                await createTestServiceConsumer(admin, resident, organization, {
+                    accountNumber,
+                })
+            })
+
+            test('can\'t create', async () => {
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await createTestMarketPriceScope(client, price, property)
+                })
+            })
+
+            test('can\'t update', async () => {
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await updateTestMarketPriceScope(client, 'id')
+                })
+            })
+
+            test('can\'t delete', async () => {
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await MarketPriceScope.delete(client, 'id')
+                })
+            })
+
+            test('can read MarketPriceScope of his organization', async () => {
+                const [obj] = await createTestMarketPriceScope(admin, price, property)
+
+                const objs = await MarketPriceScope.getAll(client, {}, { sortBy: ['updatedAt_DESC'] })
+                expect(objs).toHaveLength(0)
+            })
```

Comment:
It's good to test that each resident can read scopes linked to their property.
Also, a test about default price scopes.
```suggestion
                expect(objs).toHaveLength(0)
            })
            
            test('each of two residents can read scopes linked to their property', async () => {})
            
            test('both residents from the same organization can read default price scope', async () => {})
            
            test('residents from different organizations can read default price scopes of their organizations', () => {})
```